src/gsm/gsm.c

```diff
@@ -38,7 +38,7 @@ bool gsm_ping_modem(struct serial_buffer *sb)
 
         serial_buffer_reset(sb);
         serial_buffer_append(sb, "AT");
-        const size_t count = cellular_exec_cmd(sb, READ_TIMEOUT, msgs,
+        const size_t count = cellular_exec_cmd(sb, READ_TIMEOUT * 2, msgs,
```

Comment:
Why not just increase READ_TIMEOUT and not have unexpected behavior?

Reply:
I think that's  good idea. fixed.